ontopy/excelparser.py

```diff
@@ -304,8 +307,12 @@ def create_ontology_from_pandas(  # pylint:disable=too-many-locals,too-many-bran
                     if force is True:
                         pass
                     else:
+                        warnings.warn(
```

Comment:
This warning can be removed since we are already raising an exception with the same context below.